tests/robot-tests/tests/libs/common.robot

```diff
@@ -720,9 +720,9 @@ user checks nth breadcrumb contains
     user checks element should contain    css:[data-testid="breadcrumbs--list"] li:nth-child(${num})    ${text}
 
 user checks page contains other release
-    [Arguments]    ${other_release_title}
+    [Arguments]    ${other_release_title}    ${wait}=${timeout}
     user checks page contains element
-    ...    xpath://li[@data-testid="other-release-item"]/a[text()="${other_release_title}"]
+    ...    xpath://li[@data-testid="other-release-item"]/a[text()="${other_release_title}"]    ${wait}
```

Comment:
If you want to add a wait here, you'll want to use the `user waits until page contains element` and change the keyword name too so it reflects the fact that it waits rather than instantly checks.

That said, I'm not sure any test suite is using a wait argument with `user checks page contains other release`? If not, then I'd just remove the wait argument.

Reply:
There were a couple of test suites using this keyword. I've changed the keyword to reflect the changes I've made to: `user waits until page contains other release`